Add bash script for extracting client files

Teppic1 opened the pull request:
**Changes proposed:**

Simple bash script to automate extracting/copying client files on a Linux/Unix system.

It follows the instructions on the wiki but additionally copies camera files if found. It includes a few simple checks.

**Target branch(es):** 3.3.5

**Tests performed:** (Does it build, tested in-game, etc.)

I was able to extract and copy all files from my client.

Teppic1 committed:
Add bash script for extracting client files

Simple bash script to automate extracting/copying client files on a Linux system.

It follows the instructions on the wiki but additionally copies camera files if found. It includes a few simple checks.

Faq commented:
# 22779

jackpoz commented on contrib/extractor.sh:
```diff
@@ -0,0 +1,180 @@
+#!/bin/bash
+
+# This is where the WoW client files are located
+CLIENT_DIR=/mnt/disk/client
```
I think it's better to ask for the paths in this script

Teppic1 replied:
Updated.

Teppic1 committed:
Update extractor.sh

Client and server directories are now requested unless already set as environment variables.

Teppic1 committed:
Fix client check

illfated commented on contrib/extractor.sh:
```diff
@@ -0,0 +1,184 @@
+	if [[ ! -x $i ]]
+	then
+		echo "Invalid executable: $i"
+		echo "Check tools are installed correctly"
```
suggestion (for better understanding & readability in proper English):
```suggestion
		echo "Check that the extraction tools are installed correctly"
```

Teppic1 replied:
This is normal shell script error language, it looks a bit weird if you put errors into full normal English :)

Teppic1 committed:
Update contrib/extractor.sh

Clearer error

Co-Authored-By: Trond B. Krokli <38162891+illfated@users.noreply.github.com>

jackpoz commented:
Thanks for the PR :)
